utils/fakeuname/Makefile

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2022 Sergey V. Lobanov <sergey@lobanov.in>
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fakeuname
+PKG_RELEASE:=$(COMMITCOUNT)
+PKG_LICENSE:=GPL-2.0-or-later
+
+PKG_MAINTAINER:=Sergey V. Lobanov <sergey@lobanov.in>
+
+HOST_BUILD_DEPENDS=linux
```

Comment:
huh?

Reply:
Yes, HOST_BUILD_DEPENDS=linux might be remove. Will do

Reply:
done (HOST_BUILD_DEPENDS=linux removed)